### DISCUSSION TITLE
GTK-doc instead of Markdown?

### DISCUSSION THREAD

**Discussion by ghost** [harfbuzz/harfbuzz#3261] [Ideas] [answered]
Is their any particular reason why GTK-doc is used instead of Markdown for the docs/ directory? Sounds like its ripe for a migration.

Tagging in @n8willis

**Comment by n8willis** [comment, marked as answer]
The reason is that the project originated from GTK+ land, and with that came a lot of the conventions, infrastructure, & tooling — in fact, long before Markdown had any significant fan base; only over time has HarfBuzz taken over the remainder of the non-GTK world.

**Comment by n8willis** [comment]
Migrating the documentation isn't as simple as changing the formatting; the build system that converts it into the final published form is arguably more important (e.g., there's nothing wrong with using plain-text as the format, so long as the build generates the desired output).

Certainly you should feel free to explore ideas & strategies there. But there's more to that than mere "Markdown is newer than GTK-doc so let's find some way to lug everything into $SomethingMarkdown." That is to say, "we're using Markdown" isn't a goal; it's ancillary to "the docs build-system works well for each of the various docs segments."
